app/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -39,3 +39,29 @@ $sizes: (
 .font-weight-semibold {
   font-weight: 600;
 }
+
+.positive-significance {
+  color: green;
+  font-weight: bold;
+}
+
+.negative-significance {
+  color: red;
+  font-weight: bold;
+}
+
+.positive {
+  background-color: green;
+}
+
+.negative {
+  background-color: red;
+}
+
+.neutral {
+  background-color: grey;
+}
```

Comment:
I think we can also use Bootstrap's classes here: https://getbootstrap.com/docs/4.0/utilities/colors/#background-color

Reply:
For the css here, I used the bootstrap classes by extending them in this file. I didn't apply them directly because I didn't want to change how the javascript code references `significance` to decide which css to use

Reply:
Ah OK, `@extend` seems fine. :D